src/pages/Contributions/index.tsx

```diff
@@ -44,8 +44,8 @@ const Contributions: SFC = ({className}) => {
               <S.Td>{contribution.user.username}</S.Td>
               <S.Td>{contribution.github_user.github_username}</S.Td>
               <S.Td>{contribution.repo.name}</S.Td>
-              <S.Td>{contribution.issue.title}</S.Td>
-              <S.Td>{contribution.pull.title}</S.Td>
+              <S.Td>{contribution.issue?.title}</S.Td>
+              <S.Td>{contribution.pull?.title}</S.Td>
```

Comment:
Only added the operator for issue and pull since these 2 fields are optional. Rest are mandatory.

ps: I'll look if I can find some more side-effects, but if you come across any, please let me know. Thanks :)